### ISSUE TITLE
Item duping with Danks and Quark Inventory

### ISSUE THREAD

**Issue by VoltaicDesert** [Iskallia/Vault-public#834]
**Describe the bug**
by using a hotkey to extract items from a dank, items which have been locked in a dank can be duplicated

**To Reproduce**
1. Place an item in a dank (must be only one item in the slot)
2. Lock the item with Ctrl + left click
3. In the controls menu, bind 'Extract items' from Quark Inventory to a button
4. Open the dank with the item you wish to dupe, press the extract items button
5. The locked item will stay in its slot and also be added to your inventory

**Expected behavior**
Either the item doesn't move, or moves and doesn't leave a copy behind.

**Info**
- World Type: Single Player
- The_Vault Version 1.12.4

**Additional Info**
I used Caps lock as the extract key, and a dank 3 as the dank.

**Comment by flamingram** [comment]
i just tried and it doesent work im in singleplayer and have same ver weird

**Comment by Alexander3a** [comment]
yes because your need to enable extract items in the quake config first
